pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -724,7 +739,18 @@ func (gce *GCECloud) EnsureLoadBalancer(clusterName string, apiService *api.Serv
 			createInstances = createInstances[:maxTargetPoolCreateInstances]
 		}
 
-		if err := gce.createTargetPool(loadBalancerName, serviceName.String(), gce.region, createInstances, affinityType); err != nil {
+		// Create health checks for this target pool to pass to createTargetPool for health check links
+		hc := []*compute.HttpHealthCheck{}
+		if path, healthCheckNodePort := getServiceHealthCheckPathPort(apiService); path != "" {
```

Comment:
should it be / or /healthz ?  That gives us room for other thrid-party indicators there